babel.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
+    plugins: ['emotion'],
```

Comment:
double quotes :)
not cirtical at all, but lets keep a standard or make sure prettier goes over this file as well.